### DISCUSSION TITLE
ApprovalBundle sent an invalid submitted url

### DISCUSSION THREAD

**Discussion by pylearn2030** [kimai/kimai#5091] [Plugins]
Dear,
I am using ApprovalBundle plugin to approve the timesheets but I got the approval emails with message

"The page you were on is trying to send you to an invalid URL (http:///en/approval/week_by_user?user=2&date=2024-02-2).
 If you do not want to visit that page, you can return to the previous page"

How to correct this url? Please help!
Thank you for your help!
Py.

**Comment by kevinpapst** [comment]
Post it here: https://github.com/KatjaGlassConsulting/ApprovalBundle/

**Reply by pylearn2030** [reply]
Thank you!
